projects/eval-adxl355-pmdz/src/main.c

```diff
@@ -7,38 +7,77 @@
 #include "no-os/delay.h"
 #include "no-os/util.h"
 #include "no-os/print_log.h"
+
+#ifdef STM32_PLATFORM
 #include "stm32_spi.h"
 #include "stm32_gpio.h"
 #include "stm32_uart.h"
 #include "stm32_uart_stdio.h"
+#endif
+
+#ifdef MAXIM_PLATFORM
+#include "spi_extra.h"
+#include "gpio_extra.h"
+#include "maxim_uart.h"
+#include "maxim_stdio.h"
+#endif
+
 #include "adxl355.h"
 
-#define SPI_DEVICE_ID	 1
+#ifdef STM32_PLATFORM
+#define UART_ID			5
+#define SPI_DEVICE_ID	 	1
 #define SPI_CS			15
 #define SPI_CS_PORT		GPIOA
+#endif
+
+#ifdef MAXIM_PLATFORM
+#define UART_ID			1
+#define SPI_DEVICE_ID		0
+#define SPI_CS			0
+#endif
 
+#ifdef STM32_PLATFORM
 extern UART_HandleTypeDef huart5;
+#endif
 
 int main ()
 {
 	struct uart_desc *uart;
 	struct adxl355_dev *adxl355;
-
+	struct spi_platform_ops *spi_ops;
 	int ret;
-	stm32_init();
 
+#ifdef STM32_PLATFORM
+	stm32_init();
+	HAL_Init();
+	SystemClock_Config();
```

Comment:
These lines are not needed anymore. stm32_init is performing these actions instead.